--- a/chrome/browser/signin/signin_util_win.cc
+++ b/chrome/browser/signin/signin_util_win.cc
@@ -273,8 +273,6 @@ void SigninWithCredentialProviderIfPossi
   // This flow is used for first time signin through credential provider. Any
   // subsequent signin for the credential provider user needs to go through
   // reauth flow.
-  if (profile->GetPrefs()->GetBoolean(prefs::kSignedInWithCredentialProvider))
-    return;
 
   std::wstring cred_provider_gaia_id;
   std::wstring cred_provider_email;
